AUTHORS

```diff
@@ -65,6 +65,8 @@ Johannes Liebermann                     johanan.liebermann@gmail.com
 John Fastabend                          john.fastabend@gmail.com
 Jomen Xiao                              jomenxiao@gmail.com
 Jonathan Davies                         jpds@protonmail.com
+Jon Davies                              jpds@protonmail.com
+Joshua                                  joshroppo@gmail.com
 Joshua Roppo                            joshroppo@gmail.com
```

Comment:
Do we fix issues like these or do we usually leave it as-is?

Reply:
We usually fix them up. In v1.7 we have a .mailmap file that resolves most of these, not sure whether we can use the same in v1.6 branch or if there's some other magic somewhere. But let's make them consistent.

Reply:
(In this case I think it's just dropping the new lines)